scripts/google/covidharvard/StatisticalVariable.mcf

```diff
@@ -0,0 +1,27 @@
+Node: dcid:HarvardCOVID19CumulativeCases
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
```

Comment:
Should be MedicalCondition here.

You could do MedicalTest for the cases, but then you would change medicalStatus: dcs:ConfirmedCase to testResult: dcs:Positive. Let's bring this up in today's meeting.